From 25326a39a51996f37ef8c3cb976fe31c041a2edd Mon Sep 17 00:00:00 2001
From: James Falcon <james.falcon@canonical.com>
Date: Fri, 12 Aug 2022 10:52:50 -0500
Subject: [PATCH] Return cc_set_hostname to PER_INSTANCE frequency (#1651)

In 96eb95a the frequency was inadvertently changed to ALWAYS due to
the documentation being incorrect. The code and documentation should
now align to be PER_INSTANCE.

LP: #1983811
---
 cloudinit/config/cc_set_hostname.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cloudinit/config/cc_set_hostname.py b/cloudinit/config/cc_set_hostname.py
index 2a4c565f70..3ea9e4ed8e 100644
--- a/cloudinit/config/cc_set_hostname.py
+++ b/cloudinit/config/cc_set_hostname.py
@@ -14,9 +14,9 @@
 from cloudinit.atomic_helper import write_json
 from cloudinit.config.schema import MetaSchema, get_meta_doc
 from cloudinit.distros import ALL_DISTROS
-from cloudinit.settings import PER_ALWAYS
+from cloudinit.settings import PER_INSTANCE
 
-frequency = PER_ALWAYS
+frequency = PER_INSTANCE
 MODULE_DESCRIPTION = """\
 This module handles setting the system hostname and fully qualified domain
 name (FQDN). If ``preserve_hostname`` is set, then the hostname will not be
